bin/start

```diff
@@ -95,10 +98,19 @@ if [[ "$PROD" == "1" ]]; then
 else
     CMD="$CMD -f docker-compose.override.yaml"
 fi
+
 CMD="$CMD up"
+
 if [[ "$BUILD" == "1" ]]; then
     CMD="$CMD --build"
 fi
 
+if [[ "$LOG" == "1" ]]; then
+    CMD="$CMD -d"
+    LOG_CMD="docker-compose logs -f -t | >> NEXT_output.log 2>> NEXT_error.log"
```

Comment:
We should add .log files to .gitignore. It already appears the .log files are in .gitignore, but it appears .log is currently under Django settings.